MANIFEST.in

```diff
@@ -1,16 +1,110 @@
-exclude .cirrus.yml
-exclude .coveragerc
+# A generic template for "MANIFEST.in" files in Scitools repos.
+# NOTE: most of this can't be duplicated as-is into any given repo, but suggests
+#     guidelines + structure.
+# For the syntax of "MANIFEST.in" files, see:
+#     https://setuptools.pypa.io/en/latest/userguide/miscellaneous.html#using-manifest-in
+
+# General principles
+#     - enable user to build locally, as well as in CI
+#     - make it ignore temporary files generated by dev processes, e.g. coverage checks
+#     - encode typical decisions, e.g. whether we package docs, requirements etc
+
+#---------
+# SECTION: main code sources
+#
+recursive-include src *.py *.nc *.txt
+    # principles:
+    #    - *no* "prune" command is used
+    #    - use "include-recursive", by relevant file extensions
+    # hints:
+    #     - top-level dir is typically "src", but maybe "lib" or <package-name>
+    #     - default package rules mean we generally don't actually *need* a statement
+    #         - but it's better to be explicit
+    #     - extension filetypes are typically sources (*.py)
+    #         - might also need testdata files, e.g. *.nc, *.npy *.npz
+    #         - also possibly non-python, e.g. *.pyx for Cython
+
+
+#---------
+# SECTION: docs
+prune docs
+# and if required ... include ONLY docs source files.
+# E.G. recursive-include docs *.rst *.inc
+    # principles:
+    #    - use prune to ignore everything (optionally: except the docs sources)
+    # hints:
+    #    - most projects don't want to package the docs, but some currently do (cf-units).
```

Comment:
SciTools/.github#181 makes changes to the docs entry, if you wanted to align with that now?